packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -433,7 +439,10 @@ const client =
                 // Must be named "client". See - https://www.npmjs.com/package/webpack-hot-server-middleware#usage
                 name: CLIENT,
                 // use source map to make debugging easier
-                devtool: mode === development ? 'source-map' : false,
+                devtool:
+                    mode === development || process.env.PWA_KIT_SOURCE_MAP === 'true'
```

Comment:
Would it be better if we had some internal variable that is true when either `process.env.PWA_KIT_SOURCE_MAP` or `process.env.PWA_KIT_SSR_SOURCE_MAP` is true? That way, environments using the old `process.env.PWA_KIT_SSR_SOURCE_MAP` will get the new changes as well.

Reply:
currently there is one place that uses `PWA_KIT_SSR_SOURCE_MAP`, which is for server bundle. Due to the name has SSR in the name, it does not make sense to put it together with a variable for both client and server. Hence I created a new variable that can be used for both and looking to deprecate the old one.